src/components/calcite-shell-center-row/calcite-shell-center-row.e2e.ts

```diff
@@ -24,6 +24,19 @@ describe("calcite-shell-center-row", () => {
       }
     ]));
 
+  it("should not error when there is no action-bar", async () => {
```

Comment:
@jcfranco could we make this test reusable like the `renders` test? I don't think any components should throw a console.error.

Reply:
Looks like there's already a way to fail on console error:

```ts
it("should not error when there is no action-bar", async () =>
  await newE2EPage({
    html: "<calcite-shell-center-row></calcite-shell-center-row>",
    failOnConsoleError: true
  }));
```

Reply:
Oh nice. Ill test it out. Can we turn it on by default?

Reply:
Hmm... I don't see a way to enable it globally.

Reply:
Can we maybe add this to the renders test?